docs/General-Workflow.md

```diff
@@ -115,43 +122,43 @@ The linker script links
 - `.data` (only) from the .o built from `energy_orb_wave.data.s`
 - `.rodata` (only) from the .o built from `energy_orb_wave.rodata.s`
 
-.data (respectively .rodata) is not linked from the .o built from `energy_orb_wave.c`, because the subsegments include segments with the `data` (respectively `rodata`) segment type
-
 # 4 Decompile data
 
-Move (decompile) data/rodata to the .c, using structs or relying on strings used in the code, or other things
+TODO: `mention migrate_rodata_to_functions: yes` functionality.
```

Comment:
Btw `migrate_rodata_to_functions` is turned on by default on splat, not sure what you mean on this TODO